src/view/document.js

```diff
@@ -256,6 +256,17 @@ export default class Document {
 	}
 
 	/**
+	 * Focuses document. It will focus {@link engine.view.EditableElement EditableElement} that is currently having
+	 * selection inside.
+	 *
+	 * @fires engine.view.Document#focus
+	 */
+	focus() {
+		this.fire( 'focus' );
```

Comment:
But, if it is too tricky to provide same interface, we could try to set `isFocus` and call `render`, so let renderer to provoke native `focus` event.